### PR TITLE
Escape span method name

### PR DESCRIPTION
**Pull request by wwbmmm**
### What problem does this PR solve?

Issue Number:

Problem Summary:

### What is changed and the side effects?

Changed:

Side effects:
- Performance effects(性能影响):

- Breaking backward compatibility(向后兼容性): 

---
### Check List:
- Please make sure your changes are compilable(请确保你的更改可以通过编译).
- When providing us with a new feature, it is best to add related tests(如果你向我们增加一个新的功能, 请添加相关测试).
- Please follow [Contributor Covenant Code of Conduct](../../master/CODE_OF_CONDUCT.md).(请遵循贡献者准则).

### CODE REVIEW

File: src/brpc/builtin/rpcz_service.cpp [hunk 1]
```diff
@@ -56,6 +56,22 @@ inline std::ostream& operator<<(std::ostream& os, const Hex& h) {
     return os;
 }
 
+static std::string EscapeXSS(const std::string& data) {
```
Comment (chenBright) [inline src/brpc/builtin/rpcz_service.cpp:59]: https://github.com/apache/brpc/blob/72036f14cc4f25843fcc324599c6041d2cd30d38/src/brpc/controller.cpp#L1381-L1393

WebEscape已经实现了同样的功能，应该不用重新实现？

**Comment by gydong** [comment]
LGTM

File: src/brpc/builtin/rpcz_service.cpp (hunk 1 above)
Comment (wwbmmm) [reply src/brpc/builtin/rpcz_service.cpp]: 有道理，已改

**Comment by chenBright** [comment]
LGTM

**Pull request merged**